proto/beacon/p2p/v1/messages.proto

```diff
@@ -8,6 +8,43 @@ import "proto/eth/v1alpha1/beacon_block.proto";
 import "google/protobuf/timestamp.proto";
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
+message Hello {
+  bytes fork_version = 1 [(gogoproto.moretags) = "ssz-size:\"4\""];
+  bytes finalized_root = 2 [(gogoproto.moretags) = "ssz-size:\"32\""];
+  uint64 finalized_epoch = 3;
+  bytes head_root = 4 [(gogoproto.moretags) = "ssz-size:\"32\""];
+  uint64 head_slot = 5;
+}
+
+message Goodbye {
+  Reason reason = 1;
```

Comment:
Filed https://github.com/prysmaticlabs/go-ssz/issues/66 to support int32 fields in ssz.